### PR TITLE
Some extra ignores

### PR DESCRIPTION
**Pull request by xbauch**
no functional change.

### CODE REVIEW

**Comment by xbauch** [comment]
@hannes-steffenhagen-diffblue here: https://github.com/martin-cs/ASVAT/issues/95 .. not sure it should be included in the PR/commit description, since it's in ASVAT.

File: experiments/golden-results/StratoX-summary.txt [hunk 1]
```diff
@@ -245,7 +220,7 @@ Nkind: N_Pragma
 --
 Occurs: 1 times
 Calling function: Do_Private_Type_Declaration
-Error message: Abstract type declaration unsupported
+Error message: Tagged type declaration unsupported
```
Comment (martin-cs) [inline experiments/golden-results/StratoX-summary.txt]: Not sure I understand what is going on here.  Can you track down the code that triggers this?

File: experiments/golden-results/muen-summary.txt [hunk 2]
```diff
@@ -113,6 +93,11 @@ Calling function: Do_Aggregate_Literal
 Error message: Unhandled aggregate kind
 Nkind: N_Aggregate
 --
+Occurs: 3 times
```
Comment (martin-cs) [inline experiments/golden-results/muen-summary.txt]: Why do these appear?

**Comment by xbauch** [comment]
@hannes-steffenhagen-diffblue I've added a longer commit message and some comments in the code reasoning why it's ok to ignore those. (The comments are effectively martin's from the above linked discussion.)

**Comment by chrisr-diffblue** [comment]
@hannes-steffenhagen-diffblue - As this has two approving reviews I've merged this using admin privileges, but lets continue the discussion about the rational for this offline, and if necessary we can always revert this PR.

**Pull request merged**